Resources/Prototypes/Catalog/Cargo/cargo_fun.yml

```diff
@@ -28,6 +28,16 @@
   category: Fun
   group: market
 
+- type: cargoProduct
+  id: FunParty
+  icon:
+    sprite: Objects/Consumable/Food/Baked/cake.rsi
+    state: birthday
+  product: CrateFunParty
+  cost: 500
```

Comment:
this should be a bit more because of the food and drink, maybe 1000

Reply:
sure, its more in line with the other fun crates that way too